app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardGeneralStep/MappingWizardGeneralStep.js

```diff
@@ -1,8 +1,17 @@
 import React from 'react';
 import { Field } from 'redux-form';
-import { required } from 'redux-form-validators';
+import { required, length } from 'redux-form-validators';
 import { Form } from 'patternfly-react';
 import { FormField } from '../../../../../common/forms/FormField';
+import {
+  MAX_LENGTH_NAMES,
+  MAX_LENGTH_DESCRIPTIONS,
+  nameHelp,
+  descriptionHelp,
+  nameRequired,
+  nameTooLong,
+  descriptionTooLong
+} from '../../../../../../../../common/constants'; // Oh my
```

Comment:
funny funny.. sooooooo these are all related... used in this one and only file... what if there was an optional bundled export (instead of each an every one cuz that could get insane long run)

Reply:
That's true. Haha. Started this pattern when there were only a few and it kinda got away from me. I'll clean that up.